core/src/test/java/tech/tablesaw/table/TableTransposeTest.java

```diff
@@ -0,0 +1,193 @@
+package tech.tablesaw.table;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+import static tech.tablesaw.TableAssertions.assertTableContents;
+
+import org.junit.jupiter.api.Test;
+import tech.tablesaw.api.BooleanColumn;
+import tech.tablesaw.api.DoubleColumn;
+import tech.tablesaw.api.FloatColumn;
+import tech.tablesaw.api.IntColumn;
+import tech.tablesaw.api.LongColumn;
+import tech.tablesaw.api.StringColumn;
+import tech.tablesaw.api.Table;
+
+public class TableTransposeTest {
+
+  @Test
+  void transposeDoubles() {
+    StringColumn label = StringColumn.create("label").append("row1").append("row2").append("row3");
+    DoubleColumn value = DoubleColumn.create("value1").append(1.0).append(1.1).append(1.2);
+    DoubleColumn value2 = DoubleColumn.create("value2").append(2.0).append(2.1).append(2.2);
+
+    Table testTable = Table.create("Data");
```

Comment:
I think you can do

    Table.create("Data", label, value, value2)

Reply:
Agree. Have changed it throughout the test